src/sidebar/components/excerpt.js

```diff
@@ -113,9 +109,8 @@ function Excerpt({
       <div className="excerpt__content" ref={contentElement}>
         {children}
       </div>
-      {/* FIXME-A11Y */}
-      {/* eslint-disable-next-line jsx-a11y/no-static-element-interactions, jsx-a11y/click-events-have-key-events */}
       <div
+        role="presentation"
```

Comment:
This really felt correct here rather than trying to make this into a button or leaving the lint comment. This div is meaningless for screen readers because.

1. It can read the text without expanding it in the first place
2. There already a More/Less button that duplicates the click event here.

So this really feels like a feature for mouse users only.

Reply:
I don't agree that this qualifies for `presentation` role, alas. I can see that it might be tricksy to convert this to a button, but using `presentation` role here feels like it violates the spirit of that attribute.

Reply:
I'm not sure what this buys us over leaving the lint exception in place—I feel like it adds to future tech debt and potential confusion. But we all know my opinions about introducing too many `aria` attributes, so if I'm overruled, so be it...